### ISSUE TITLE
[Discussion] Is this project necessary

### ISSUE THREAD

**Issue by ethanx94** [Harmmmm/MKDX-English-Mod#1]
I’ve been studying this game since I moved to a town with...several arcades. Does the “Grand Prix” mode of Mario Kart “GP” DX really not exist in the US version? Every place I tried went to a game over screen after just one race. At first, I thought it was just a dip switch or toggle the cabinet owner set to keep people shoveling in quarters. I guess without a proper dump no one can say for sure but at after trying 3 different arcades with the same result, I’ve got to wonder can anyone confirm the USA version really has no Grand Prix mode?

**Comment by yegglet** [comment]
Yes! I would like to know if it's possible to use this patcher on a bonafide cabinet. I recently acquired a US cabinet, and it is dismal not being able to play Grand Prix mode. Even though it's IN the trailer.

**Comment by GeminiKing2006JC** [comment]
> Yes! I would like to know if it's possible to use this patcher on a bonafide cabinet. I recently acquired a US cabinet, and it is dismal not being able to play Grand Prix mode. Even though it's IN the trailer.

Wait if you have the USA Version of the game maybe you can dumped it to the public for Teknoparrot so we can all play it.
